src/Profile.js

```diff
@@ -0,0 +1,28 @@
+const styles = {
+  backgroundColor: "lightGrey",
+  fontFamily: "sans-serif",
+  boxShadow: "5px 10px #888888",
+  width: "40vw",
+  margin: "0 auto",
+  padding: "5%",
+};
+
+export function Profile({ profile, setProfile }) {
+  if (profile.shows) {
+    return (
+      <div style={styles}>
+        <h1>{profile.fullName}</h1>
+        <p>{profile.bio}</p>
+        <p>{profile.profession}</p>
+        <img src={profile.imgSrc} alt="Hannah"></img>
+      </div>
+    );
+  } else {
```

Comment:
No need for the `else` statement here; if the first condition is fulfilled it will exist the component because of the `return` statement anyway, thus you could get rid of the the nesting:

```jsx
export function Profile({ profile, setProfile }) {
  if (profile.shows) {
    return (
      <div style={styles}>
        <h1>{profile.fullName}</h1>
        <p>{profile.bio}</p>
        <p>{profile.profession}</p>
        <img src={profile.imgSrc} alt="Hannah"></img>
      </div>
    );
  
  return (
    <button onClick={() => setProfile({ ...profile, shows: true })}>
      Click Me Buddy
    </button>
  );
}
```

You could also make use of the ternary operator:

```jsx
export function Profile({ profile, setProfile }) {
  return profile.shows ? (
    <div style={styles}>
      <h1>{profile.fullName}</h1>
      <p>{profile.bio}</p>
      <p>{profile.profession}</p>
      <img src={profile.imgSrc} alt="Hannah"></img>
    </div>
  ) : (
    <button onClick={() => setProfile({ ...profile, shows: true })}>
      Click Me Buddy
    </button>
  );
}
```